activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -60,6 +60,12 @@ def test_should_include_other_with_exclusive_end
     assert((1..10).include?(1...11))
   end
 
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.6.0")
+    def test_include_with_endless_range
+      assert((1..).include?(2))
```

Comment:
To prevent this test from breaking CI on Rails 2.5, you can use `eval` to defer parsing until runtime:

```suggestion
      assert(eval("1..").include?(2))
```

🙇  @sikachu for coming up with this solution! https://github.com/rails/rails/pull/34906#discussion_r247001824